Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,7 +115,45 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
-    activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
+
+    // Parse command line options
+    string requestedURL;
+
+    for (auto option : options) {
+        bool foundOption = false;
+        __debugbreak();
```

Comment:
Why do you want to __debugbreak here?. is this intentional?

Reply:
resolved in https://github.com/WebKit/WebKit/pull/5154/commits/ec72e16864b4383d8497aac77fa097414823a309